### ISSUE TITLE
deploy source code to app service

### ISSUE THREAD

**Issue by rupesh15203** [Azure-Samples/azure-search-openai-demo#626]
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue is for a: (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [ ] feature request
- ['x'] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
>Considering there is already resources (Azure OpenAI, Form Recognizer, Search Service, Storage) present in resource group. New app service has been created through portal. 

### Any log messages given by the failure
>

### Expected/desired behavior
>Can we use "azd deploy" to deploy this application in newly created app service by setting required parameters in app. 

### OS and Version?
> Windows 7, 8 or 10. Linux (which distribution). macOS (Yosemite? El Capitan? Sierra?)

### azd version?
> azd version 1.2.0

### Versions
>

### Mention any other details that might be useful

> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

**Comment by d-paulus** [comment]
You can **(a)** use `azd up` to create a new app service and deploy the app to it, or **(b)** use `azd deploy` to deploy to an existing app service. Both approaches allow reusing existing resources.

**(a)**
Comment out the creation of all other resources in `infra/main.bicep` and only keep creation of app service.

**(b)**
Change values of environment variables in `.azure/{YOUR-ENV}/.env` : _AZURE_ENV_NAME, AZURE_LOCATION, AZURE_RESOURCE_GROUP, AZURE_SUBSCRIPTION_ID, AZURE_TENANT_ID, BACKEND_URI_.

**Comment by DSgUY** [comment]
Hi, I create the azure app service and configure my gitlab with the app/backend folder. 
How I can config the Azure App Service to deploy this folder? The logs shows that the requirements.txt was not found as is searching in the root of the master and not in app/backend folder.

I tried with .yaml .deployment and also with adding Application settings in Configuration Settings.

I want to keep my repo and not have a new project or brunch only for the backend.

Thanks.

**Comment by rupesh15203** [comment]
@d-paulus, I tried to deploy it using VS code. I deployed it to app service which I wanted. I am facing same issue faced by @DSgUY. App is also not functional once it was deployed.

Hence I just deployed backend folder using CS code. With this app is deployed and somehow it has picked up the frontend also. But when I tried to do some changes in frontend, they are not reflecting in app service.

Also, I am not able to find frontend source code in container(App service>Advance tools>Kudu app>SSH).

Any idea where exactly frontend code hosted?

Thanks.
